containers/Settings.js

```diff
@@ -84,38 +82,17 @@ const sendFeedback = () => {
 };
 
 const mapDispatchToProps = (dispatch, ownProps) => ({
-  logout: () => dispatch(logout()),
   acceptInvite: () => ownProps.navigation.navigate('Invites'),
+  clearPushPermission: () => dispatch(clearPushPermission()),
   exportData,
+  locationData: () => ownProps.navigation.navigate('LocationData'),
```

Comment:
Should this be `showLocationData`?

Reply:
The function call name or the actual route name?

Reply:
I was thinking the call name